charts/jaeger/templates/jaeger-v2-agent-svc.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.allInOne.enabled -}}
```

Comment:
why do we need to rename all these files? Please keep the same names - there should be no files with `v2` in the name.